File: daily-js/moveZeroes.js
const moveZeroes = arr => {
    let zeroCounter = 0;
    const numberZero = 0;
    for (let i = arr.length -1; i >=0; i--) {
      if (arr[i] === 0) {
        zeroCounter++;
        arr.splice(i,1);
      }
    }
    for (let currentCount = 0; currentCount < zeroCounter; currentCount++) {
      arr.push(0);
    }
    return arr;
  }
  
  let x = moveZeroes([1,2,3,0,0,0,1,6,0]);
  console.log(x);

File: daily-py/move_zeroes.py
def move_zeroes(arr):
    i = 0
    temp = []
    while i < len(arr):
        if arr[i] != 0:
            temp.append(arr.pop(i))
        else:
            i += 1
    return temp + arr
            
lst = [0,0,0,1,2,3,0,0,4,0,1]
x = move_zeroes(lst)
print(x)